xlators/cluster/dht/src/dht-rebalance.c

```diff
@@ -4717,7 +4716,7 @@ gf_defrag_stop(dht_conf_t *conf, gf_defrag_status_t status, dict_t *output)
     defrag->defrag_status = status;
 
     if (output)
-        gf_defrag_status_get(conf, output);
+        gf_defrag_status_get(conf, output, log_status);
```

Comment:
NIT: with the current usage, `log_status` can be omitted and changed here to  `_gf_false` as for `_gf_true` output is NULL.
```
xlators/cluster/dht/src/dht-common.c:                    gf_defrag_stop(conf, GF_DEFRAG_STATUS_FAILED, NULL, _gf_true);
xlators/cluster/dht/src/dht-common.c:                    gf_defrag_stop(conf, GF_DEFRAG_STATUS_STOPPED, output, _gf_false);
``` 
This is NIT because having this var might be clearer and better for future usage..

Reply:
Done